src/nav/nav.ts

```diff
@@ -173,12 +183,20 @@ export class NgbNav implements AfterContentInit {
    */
   @Input() roles: 'tablist' | false;
 
+  /**
+ * Determine if the tabs will be automaticaly changed when user pushes arrow keys(by default false)
+ */
+  @Input() selectOnKeyboardEvent: boolean;
```

Comment:
Let's change the API similar to the once we have in carousel:

```ts
@Input() keyboard: boolean | 'changeWithArrows';
```

- `false` → no keyboard support whatsoever
- `true` → keyboards support if `role=tab`
- `'changeWithArrows'` → changes the tab automatically with keyboard arrow (same as `selectOnKeyboardEvent = true` before

If you have better naming suggestions for `changeWithArrows`, don't hesitate.